spaceroombas/data/state.py

```diff
@@ -93,6 +92,9 @@ def connect_sectors(self, sector1id, sector2id):
         else:
             return False
 
+    def get_sector_ids(self):
+        return list(self.__sectors)
```

Comment:
Consider using dict.keys()

Reply:
tbh, i thought i had it as list(self.__sectors.keys())